src/app/pages/BuyPage/components/Button/buy.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+
+interface BtnProps {
+  onClick: () => void;
+  disabled?: boolean;
+}
+
+interface Props extends BtnProps {
+  text: React.ReactNode;
+  className?: string;
+}
+
+const StyledButton = styled.button`
```

Comment:
Pls convert the `styled-components` into `module css`

Reply:
fixed